<robot name="exp_small_obj">
  <link name="small_obj">
    <inertial>
      <origin xyz="0.03 0.03 0.0325"/>
      <mass value="0.5" />
      <inertia  ixx="0.5" ixy="0.0"  ixz="0.0"  iyy="0.5"  iyz="0.0"  izz="0.5" />
    </inertial>
    <visual>
      <origin xyz="0.03 0.03 0.0325"/>
      <geometry>
        <cylinder length="0.065" radius="0.03"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.03 0.03 0.0325"/>
      <geometry>
        <cylinder length="0.065" radius="0.03"/>
      </geometry>
    </collision>
  </link>

  <gazebo reference="small_obj">
    <material>Gazebo/Green</material>
     <mu1>1000</mu1>
     <mu2>1000</mu2>
  </gazebo>
</robot>
